web/pgadmin/browser/server_groups/servers/roles/templates/roles/sql/default/dependents.sql

```diff
@@ -10,7 +10,7 @@ ORDER BY 1, 2
 {% endif %}
 
 {% if fetch_dependents %}
-SELECT cl.relkind, COALESCE(cin.nspname, cln.nspname) as nspname,
+SELECT CAST(cl.relkind as text), COALESCE(cin.nspname, cln.nspname) as nspname,
```

Comment:
or try `cl.relkind::text` which is used mostly.

Reply:
Also test on other PG server verions.

Reply:
Tested on PG 12 and above.